src/DIRAC/FrameworkSystem/private/standardLogging/Logging.py

```diff
@@ -161,82 +164,67 @@ def registerBackends(self, desiredBackends, backendOptions=None):
                                  list of the possible values: ['stdout', 'stderr', 'file']
         :param backendOptions: dictionary of different backend options. Example: FileName='/tmp/log.txt'
         """
-        for backendName in desiredBackends:
-            self.registerBackend(backendName, backendOptions)
+        return S_ERROR("This method is deprecated, please use registerBackend() instead.")
```

Comment:
This is part of the `private` namespace so probably nobody should be using this function, but if it has to be removed the following points apply:

Nothing checks the return value of this function. So better `raise NotImplementedError`, or even better add the `deprecated` decorator that is somewhere in DIRAC before decommissioning this function. And if it is just deprecated it still needs to do what it did before.

Reply:
This method can actually be used via `gLogger` (e.g. `gLogger.registerBackends()`). I added the `deprecated` decorator and keep the method as it was, thanks.